src/db/spanner/batch.rs

```diff
@@ -152,107 +152,6 @@ pub fn delete(db: &SpannerDb, params: params::DeleteBatch) -> Result<()> {
     Ok(())
 }
 
-pub fn _audit_commit_err<T>(
```

Comment:
pretty sizable delete here. IIUC, we're no longer auditing if the GRPC call fails, correct?

Reply:
We'll still get the normal sentry reporting for it, all this code predated the sentry capturing w/ tags (so some of it's now duplicated).

The mutation counts were nice to have to diagnose the original issue, but are no longer necessary